astropy/io/fits/tests/test_table.py

```diff
@@ -1758,6 +1758,10 @@ def test_mismatched_tform_and_tdim(self):
         pytest.raises(VerifyError, fits.Column, name='a', format='2I',
                       dim='(2,2)', array=arra)
 
+    def test_tdim_of_size_one(self):
```

Comment:
If you don't mind, please add a docstring to this test like the one below it linking back to this issue.